tutorials/plugins/editor/inspector_plugins.rst

```diff
@@ -179,8 +231,72 @@ followed by ``set_bottom_editor()`` to position it below the name.
         # Update the control with the new value.
         updating = true
         current_value = new_value
-        property_control.text = "Value: " + str(current_value)
+        refresh_control_text()
         updating = false
+    
+    func refresh_control_text():
+        property_control.text = "Value: " + str(current_value)
+
+ .. code-tab:: csharp
+
+    // RandomIntEditor.cs
+    using Godot;
+
+    public class RandomIntEditor : EditorProperty
+    {
+        // The main control for editing the property.
+        private Button _propertyControl = new Button();
+        // An internal value of the property.
+        private int _currentValue = 0;
+        // A guard against internal changes when the property is updated.
+        private bool _updating = false;
+
+        public RandomIntEditor()
+        {
+            // Add the control as a direct child of EditorProperty node.
+            AddChild(_propertyControl);
+            // Make sure the control is able to retain the focus.
+            AddFocusable(_propertyControl);
+            // Setup the initial state and connect to the signal to track changes.
+            RefreshControlText();
+            _propertyControl.Connect("pressed", this, nameof(OnButtonPressed));
```

Comment:
I forgot this is `master`'s documentation. The connecting syntax has changed and should now be:

```suggestion
            _propertyControl.Pressed += OnButtonPressed;
```